### PR TITLE
[4.0] skip to show joomla token on registration

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #28567 .

### Summary of Changes
skip the display of joomlatoken when on registration

### Testing Instructions
Enable the user joomla token plugin
Enable api authentication plugin
Enable registration on the front end
Register a new user

### Expected result
no  Joomla token shown

### Actual result
Screenshot from 2020-04-06 10-40-39

### CODE REVIEW

File: components/com_users/tmpl/registration/default.php [hunk 1]
```diff
@@ -29,6 +29,9 @@
 		<?php foreach ($this->form->getFieldsets() as $fieldset) : ?>
 			<?php $fields = $this->form->getFieldset($fieldset->name); ?>
 			<?php if (count($fields)) : ?>
+				<?php if ($fieldset->name === 'joomlatoken') : ?>
```
Comment (wilsonge) [inline components/com_users/tmpl/registration/default.php]: This check needs to be done in the user plugin that renders these fields - not here

Comment (alikon) [reply components/com_users/tmpl/registration/default.php]: sounds better now ?

**Comment by wilsonge** [comment]
If we're not going to show this data on registration (which kinda seems logical until they get assigned a user group) then we should just remove it from the list of allowed context's here https://github.com/joomla/joomla-cms/pull/28584/files#diff-618b02bb14a772431ff46eee43117ffeR190

**Comment by alikon** [comment]
ok removed from the array of allowed contexts

**Comment by toivo** [comment]
I have tested this item :white_check_mark: successfully on bbab3e40d9ef054282061e6841414e0157b8abf8<br /><br />Tested successfully in beta1-dev full version 20200408 <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28584">issues.joomla.org/tracker/joomla-cms/28584</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**